server/src/app.routes.ts

```diff
@@ -43,24 +45,29 @@ router.post("/base/updatesaved", auth, debugDataLog("Base updated save"), update
 router.get("/api/bm/yardplanner/gettemplates", auth, debugDataLog("Get templates"), getTemplates);
 router.post("/api/bm/yardplanner/savetemplate", auth, debugDataLog("Saving template"), saveTemplate);
 
+
+router.post("/api/bm/neighbours/get", debugDataLog("Getting user neighbors"), async (ctx) => {
+  ctx.status = 200;
+  ctx.body = {
+    error: 0,
+  }
+})
```

Comment:
Although small in size, we should move this into it's own controller, for consistency.